cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/DefaultMetricDatasetFactory.java

```diff
@@ -70,7 +70,8 @@ public FactTable getOrCreateFactTable(int resolution) {
     String v3TableName = cConf.get(Constants.Metrics.METRICS_TABLE_PREFIX,
                                    Constants.Metrics.DEFAULT_METRIC_V3_TABLE_PREFIX) + ".ts." + resolution;
 
-    int ttl = cConf.getInt(Constants.Metrics.RETENTION_SECONDS + "." + resolution + ".seconds", -1);
+    int ttl = cConf.getInt(Constants.Metrics.RETENTION_SECONDS + resolution +
+                             Constants.Metrics.RETENTION_SECONDS_SUFFIX, -1);
```

Comment:
should not hardcode -1 as a default. It should be in cdap-default.xml

Reply:
Noticed they are already defined in cdap-default.xml, removed them